backend/dal/Helpers/Extensions/ContactExtensions.cs

```diff
@@ -59,13 +59,11 @@ private static IQueryable<Entity.Contact> GenerateCommonContactQuery(this IQuery
         /// <param name="context"></param>
         /// <param name="filter"></param>
         /// <returns></returns>
-        public static IQueryable<Entity.Contact> GenerateContactQuery(this PimsContext context, Entity.Models.ContactFilter filter)
+        public static IQueryable<Entity.PimsContactMgrVw> GenerateContactQuery(this PimsContext context, Entity.Models.ContactFilter filter)
         {
             filter.ThrowIfNull(nameof(filter));
 
-            IQueryable<Contact> query = context.Contacts.AsNoTracking()
-                .Include(c => c.Person)
-                .ThenInclude(p => p.ContactMethods);
+            IQueryable<PimsContactMgrVw> query = context.PimsContactMgrVws.AsNoTracking();
```

Comment:
database first views are generated with hasnokey(), unless that can be overriden somehow they can't be the principal for property navigation.